src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+    <header>
+        <h1>Leaderboard</h1>
+    </header>
+
+    <main class="main-container">
+        <section class="part1">
+            <div class="header1">
+                <h2>Recent scores</h2>
+                <button class="refresh-btn"><b>Refresh</b></button>
+            </div>
+
+            <div class="listOfNames">
+                <ul>
+                    <li>Name <span>:100</span></li>
+                    <li>Name <span>:20</span></li>
+                    <li>Name <span>:50</span></li>
+                    <li>Name <span>:78</span></li>
+                    <li>Name <span>:125</span></li>
+                    <li>Name <span>:77</span></li>
+                    <li>Name <span>:42</span></li>
+                </ul>
+            </div>
+        </section>
+
+        <section class="part2">
+            <h2>Add your score</h2>
+            <form class="check-in">
+                <input type="text" name="Name" maxlength="30" placeholder="Your name" required>
+                <input type="number" name="Scores" placeholder="Your scores" required>
+                <button type="submit">Submit</button>
+            </form>
+        </section>
+    </main>
+    <script src="index.js"></script>
```

Comment:
- [ ] Nice work implementing the design :clap: However it would be nice to have the two section aligned on the same row as in the provided [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), to follow the project requirement.

Reply:
I have 2 sections already please.